config/locales/petitions.en-GB.yml

```diff
@@ -11,11 +11,9 @@ en-GB:
     facets:
       public:
         - :all
-        - :open
-        - :referred
-        - :debated
-        - :not_debated
-        - :completed
+        - :collecting_signatures
```

Comment:
Can you leave in the debated ones - the feature flag switches to the alternate list so I want to be able to bring those back if required on preview